docs/Metrics/DeployTime.md

```diff
@@ -16,11 +16,20 @@ The time it takes from when a PR is merged to when it is deployed.
 
 
 ## How is it calculated?
-You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
-
+1. You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+2. Connect pr/mr to the right `deployment`
+![](/img/Metrics/pr-commit-deployment-relation.jpeg)
+   1. Use plugin `refdiff` to calculate `refs_commits_diffs` between any two consecutive deployments under the same scope,
+for example, here, we will get commit-2 and commit-3 by calculating commits_diff before deployment-1 and deployment-2,
+we can say commit-2, commit-3 and commit-4 are deployed by deployment-2
+   2. Connect pr/mr and commits_diff through merge_commit or pr_commit, for example, 
+we get pr-3 connected to commit-3
+   3. Now we can get pr-3's deploy time by finish_time of deployment-2 minus merge_time of pr-3.
```

Comment:
Please refer to [issue 3516](https://github.com/apache/incubator-devlake/issues/3516), not all commits can be mapped to PRs.